client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -50,13 +59,27 @@ export default function Dashboard(): JSX.Element {
     return <CircularProgress />;
   }
 
+  const getListsData = async () => {
+    getAllLists().then((data) => {
+      setLists(data.lists);
+    });
+  };
```

Comment:
Defining a asynchronouse arrow-function gives you the ability to do `await`. This could all be cleaned up to just two lines of code.